km_api/know_me/models.py

```diff
@@ -283,6 +203,117 @@ def __str__(self):
         return self.user.get_short_name()
 
 
+class ListContent(models.Model):
+    """
+    Model to store list content for a :class:`.ProfileItem`.
+    """
+    profile_item = models.OneToOneField(
+        'know_me.ProfileItem',
+        on_delete=models.CASCADE,
+        related_name='list_content',
+        verbose_name=_('profile item'))
+    """
+    :class:`.ProfileItem`:
+        The profile item that the list content is attached to.
+    """
+
+    class Meta(object):
+        verbose_name = _('profile item list content')
+        verbose_name_plural = _('profile item list content')
```

Comment:
The best plural I could come up with was `profile items' list content` which doesn't seem any clearer to me. [This page](https://jakubmarian.com/content-vs-contents-in-english/) seems to suggest that "content" refers to everything in a container as a single object whereas "contents" refers to all the items in a container. To me, that suggests we should either use "content" in both places, or "contents" in both places.